docs/dotnet-ubuntu.md

```diff
@@ -0,0 +1,34 @@
+# Ubuntu .NET Core Versions
+
+.NET has changed the recommended install methods for Ubuntu from 2404.
+
+This document gives an overview of these change and the impact this has on the `runner-images`.
+
+## .NET Core for Ubuntu 2004 and 2204
+
+2004 and 2024 use the [Microsoft Package repository](https://learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu-install?tabs=dotnet8&pivots=os-linux-ubuntu-2004) to install .NET deb files built and published by the .NET team.
```

Comment:
`2024` is still mentioned here, please correct this to `2404`.

Reply:
:facepalm: My bad. Thanks :heart: 
```suggestion
2004 and 2204 use the [Microsoft Package repository](https://learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu-install?tabs=dotnet8&pivots=os-linux-ubuntu-2004) to install .NET deb files built and published by the .NET team.
```